spec/integration/post_index_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :system do
+  fixtures :users, :posts, :comments
+  let(:user1) { users(:one) }
+  describe 'Index page' do
+    it "can see the user's profile picture." do
+      visit user_posts_path(user1)
+      expect(page).to have_selector("img[src='#{user1.photo}']")
+    end
+    it "can see the user's username." do
+      visit user_posts_path(user1)
+      expect(page).to have_content(user1.name)
+    end
+    it 'can see the number of posts the user has written.' do
+      visit user_posts_path(user1)
+      expect(page).to have_content(user1.post_counter)
+    end
+    it "can see a post's title." do
+      visit user_posts_path(user1)
```

Comment:
- [ ] Kindly ensure that all your tests are passing as right now the `Test that asserts you can see the post's title` is failing. Make sure you fix that.
![Screenshot from 2023-03-08 08-03-30](https://user-images.githubusercontent.com/83042742/223632766-4d3d6a04-ad72-40ea-a1ec-01e7a6fa3fa7.png)

Reply:
@tresorsawasawa I check it in my side it working please look at the below image

![image](https://user-images.githubusercontent.com/109666020/223638519-ae2132e2-1ce7-4d72-a147-f85d7caa68a2.png)

Reply:
i don't know why it's failing at your side

Reply:
Please check out it's working fine my side every integration test

![image](https://user-images.githubusercontent.com/109666020/223655577-8c7b3dee-5f06-41da-bedb-37d20ccf7384.png)